core/tests/protractor_desktop/voiceoverUploadFeatures.js

```diff
@@ -0,0 +1,114 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests for the functionality of the upload files
+ */
+
+var forms = require('../protractor_utils/forms.js');
+var general = require('../protractor_utils/general.js');
+var users = require('../protractor_utils/users.js');
+var workflow = require('../protractor_utils/workflow.js');
+
+var ExplorationEditorPage =
+    require('../protractor_utils/ExplorationEditorPage.js');
+
+describe('Exploration tab', function() {
+  var TEST_USERNAME = 'uploadUser';
+  var TEST_EMAIL = TEST_USERNAME + '@example.com';
+  var explorationEditorPage = null;
+  var explorationEditorMainTab = null;
+  var explorationEditorTranslationTab = null;
+  var explorationEditorSettingsTab = null;
+
+  beforeAll(function() {
+    explorationEditorPage = new ExplorationEditorPage.ExplorationEditorPage();
+    explorationEditorMainTab =
+      explorationEditorPage.getMainTab();
```

Comment:
I think this line should fit on line 37 itself right?

Reply:
Done